lib/screen/weather_screen.dart

```diff
@@ -14,10 +17,17 @@ class WeatherScreen extends StatefulWidget {
 class _WeatherScreenState extends State<WeatherScreen> {
   final _weather = YumemiWeather();
   String _weatherImage = '';
+  String _lowestTemperature = '**';
+  String _highestTemperature = '**';
 
-  void _reloadWeather() {
+  Future<void> _reloadWeather() async {
     try {
-      _weatherImage = _weather.fetchThrowsWeather('tokyo');
+      final loadJson = await rootBundle.loadString(constants.jsonFile);
+      final getWeather = _weather.fetchWeather(loadJson);
```

Comment:
[nits]
`getWeather` だとメソッドを定義しているものと勘違いしてしまうため、次のような名前のほうが分かりやすそうです。ほかに良さそうな名前があればそれでもいいかなと思います 🙆‍♂️ 

```suggestion
      final weatherJsonText = _weather.fetchWeather(loadJson);
```